### PR TITLE
add flat_map to dolly collection

### PR DESCRIPTION
**Pull request by seancookr**

### CODE REVIEW

**Comment by seancookr** [comment]
actually it would seem that sets don't respond to `#flat_map`   

http://www.ruby-doc.org/stdlib-2.2.0/libdoc/set/rdoc/Set.html

File: Gemfile.lock [hunk 1]
```diff
@@ -10,7 +10,7 @@ GIT
 PATH
   remote: .
   specs:
-    dolly (0.7.5)
+    dolly (0.7.6)
```
Comment (javierg) [inline Gemfile.lock]: conflicting version number...

**Comment by javierg** [comment]
sp... now just add a test  :)

**Comment by seancookr** [comment]
says the person with no tests on the `Dolly::Collection` Class.  I guess I can help with that

**Comment by javierg** [comment]
> says the person with no tests on the Dolly::Collection Class. I guess I can help with that

The guy who knows its flaws and ask for help ;-)

**Comment by seancookr** [comment]
:+1:

**Comment by javierg** [comment]
:shipit:

**Pull request merged**